### DISCUSSION TITLE
Can't view embedded html PDF files in vscode jupyter?

### DISCUSSION THREAD

**Discussion by MikeLemo1** [microsoft/vscode-jupyter#13769] [Questions and Answers]
I'm trying to reference an internal PDF file to view a single page from it with embedded HTML in VSCode Jupyter plugin with no luck as it just displays nothing or blank rectangles with no luck(just for reference the same method works in mkdocs)
Here is what I tried to do it with assuming the PDF file is sitting in the same folder as the .ipynb file:

`from IPython.display import IFrame

# display(HTML('<embed id="myPDF2" src = "STM32F302xD_E_MCU.pdf#page=34&zoom=150&toolbar=1&statusbar=1&viewrect=100%,100%,50%,50%" type="application/pdf" width="100%" height=800px />'))

IFrame( src = 'STM32F302xD_E_MCU.pdf#page=34&zoom=150&toolbar=1&statusbar=1', width=700, height=600)` 

Also tried: 

`%%HTML
<embed id="myPDF2" src = "STM32F302xD_E_MCU.pdf#page=34&zoom=150&toolbar=1&statusbar=1&viewrect=100%,100%,50%,50%" type="application/pdf" width="100%" height=800px />`

Any idea what can be done to help it work?

**Comment by DonJayamanne** [comment]
Thanks for filing this issue and sorry you are running into this.

Please can you open the console window using the command `Toggle Developer Tools` and check if there are any errors in the `Console Window` and the network tab.

**Reply by MikeLemo1** [reply]
Yes there you go: 

`
Failed to load https://file+.vscode-resource.vscode-cdn.net/c%3A/MyFolder/GreenEnergy/Documentation/STM32F302xD_E_MCU.pdf#page=34&zoom=150&toolbar=1&statusbar=1&viewrect=100%,100%,50%,50% as a plugin, because the frame into which the plugin is loading is sandboxed.

DevTools failed to load source map: Could not load content for https://file+.vscode-resource.vscode-cdn.net/c%3A/Users/mikes/.vscode/extensions/ms-toolsai.jupyter-renderers-1.0.15/out/client_renderer/preload.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED

DevTools failed to load source map: Could not load content for https://file+.vscode-resource.vscode-cdn.net/c%3A/Users/mikes/.vscode/extensions/ms-toolsai.jupyter-2023.5.1101742258-win32-x64/out/webviews/webview-side/ipywidgetsKernel/ipywidgetsKernel.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED

DevTools failed to load source map: Could not load content for https://file+.vscode-resource.vscode-cdn.net/c%3A/Users/mikes/.vscode/extensions/ms-toolsai.jupyter-renderers-1.0.15/out/client_renderer/builtinRendererHooks.js.map: Connection error: net::ERR_NAME_NOT_RESOLVED
`

**Reply by MikeLemo1** [reply]
Well I also found some 2 related topics:
[https://github.com/microsoft/vscode-jupyter/issues/3654](url)
[https://github.com/microsoft/vscode/issues/172867](url)
